### PR TITLE
Bug 1159177 - [Wifi] disable submit button if no ssid, r=Arthur

### PR DESCRIPTION
**Pull request by EragonJ**

### CODE REVIEW

**Comment by lightsofapollo** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=da75e545be67bc5db14b28ec50e188ff0f954312

File: apps/settings/js/modules/wifi_utils.js [hunk 1]
```diff
@@ -186,9 +187,17 @@ define(function(require) {
       };
 
       var checkPassword = function() {
+        var isSSIDInValid = function() {
```
Comment (crh0716) [inline apps/settings/js/modules/wifi_utils.js]: nit: isSSIDInvalid

File: apps/settings/js/panels/wifi_manage_networks/panel.js [hunk 2]
```diff
@@ -43,7 +43,10 @@ define(function(require) {
                 value.identity,
                 value.eap
               );
-              WifiContext.associateNetwork(network, function() {
+              WifiContext.associateNetwork(network, function(error) {
+                if (error) {
+                  DialogService.alert(error);
```
Comment (crh0716) [inline apps/settings/js/panels/wifi_manage_networks/panel.js]: It seems not a good idea to display the error message directly to users using dialog service. Is it possible to map it to an user-friendly message?

File: apps/settings/js/modules/wifi_utils.js (hunk 1 above)
Comment (EragonJ) [inline apps/settings/js/modules/wifi_utils.js]: ok !

File: apps/settings/js/panels/wifi_manage_networks/panel.js (hunk 2 above)
Comment (EragonJ) [inline apps/settings/js/panels/wifi_manage_networks/panel.js]: based on our offline discussion, let's move this into next follow-up bug.

**Comment by lightsofapollo** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=58029ae4dbc8655b4857ba16a92b777670674f6c

**Comment by autolander** [comment]
http://docs.taskcluster.net/tools/task-graph-inspector/#HBiNVytCRGKGLN17POaKGg

The pull request failed to pass integration tests. It could not be landed, please try again.

**Comment by autolander** [comment]
http://docs.taskcluster.net/tools/task-graph-inspector/#HkmbiWqeTESX5OqttFXhUA

The pull request failed to pass integration tests. It could not be landed, please try again.

**Comment by lightsofapollo** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=8e536eb843c1e2f4289e08bb555f75c4ff7693ad

**Comment by EragonJ** [comment]
Rebased to master and give it a try again ! Let's see whether we are lucky enough to get green light !

**Pull request merged**